Resources/Prototypes/Nyanotrasen/Entities/Decoration/snowman.yml

```diff
@@ -0,0 +1,34 @@
+- type: entity
+  id: Snowman
+  parent: BaseStructure
```

Comment:
A circular bounding box might be more fitting than the default box from BaseStructure. See the related comment.